### PR TITLE
CONTRIBUTING.md Tidy.exe + Au3Check.exe

### PR DESCRIPTION
**Pull request by mlipok**
Coding Guidelines should be suplemented as workflows use Au3Check.exe but contributor should check his contribution before making PR.
Code should be tidied because such code is easier to read and understand, for this reason Tidy.exe was mentioned.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 * Use CamelCase variables
 * Tabs, not spaces, with tab width set to 4
 * Follow https://www.autoitscript.com/wiki/Best_coding_practices
+* Tidy code with Tidy.exe before you create PR
+* Use Au3Check.exe for pre check code before you create PR
```
Comment (micwoj92) [inline CONTRIBUTING.md:7]: This is already in the "best coding practices" link

**Review by rcmaehl — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mlipok) [reply CONTRIBUTING.md:6]: oops. checked. 

Almost true.
I mean there is mentioned to use Au3Check but not about Tidy.exe

What you think about ?

Comment (micwoj92) [reply CONTRIBUTING.md:6]: I think that tidy step should be first step in pull request guidelines section and get rid of au3check

Comment (rcmaehl) [reply CONTRIBUTING.md:6]: Nah Au3Check should be included or it'll main au3check on the main script

Comment (micwoj92) [reply CONTRIBUTING.md:6]: I still think these should be added as first in pr guidelines, for example:
Run Tidy and Au3check on your code to make sure that it is formatted properly.

Comment (rcmaehl) [reply CONTRIBUTING.md:6]: But then it doesn't look *NICE* as a list 😆

Comment (micwoj92) [reply CONTRIBUTING.md:6]: eh ok

**Pull request merged**